itests/sector_import_simple_test.go

```diff
@@ -39,7 +39,9 @@ func TestSectorImportAfterPC2(t *testing.T) {
 	////////
 	// Start a miner node
 
-	client, miner, ens := kit.EnsembleMinimal(t, kit.ThroughRPC())
+	// We use two miners so that in case the actively tested miner misses PoSt, we still have a blockchain
+	client, miner, _, ens := kit.EnsembleOneTwo(t, kit.ThroughRPC())
+
 	ens.InterconnectAll().BeginMining(blockTime)
```

Comment:
Actually in this case BeginMiningMustPoSt should help. What I think may be happening here is that we are trying to produce the post snark, but that is blocked while the sector is Committing - in which case EnsembleOneTwo may still not be robust enough, because Committing even for 2k sectors takes a few seconds, and takes the global gpu lock, which will prevent any post from being created

Reply:
We've had _pretty_ good luck since we turned the other test to use 2 miners. I'm gonna merge this for now, but will definitely try `MustPost`ing if the issue recurs.